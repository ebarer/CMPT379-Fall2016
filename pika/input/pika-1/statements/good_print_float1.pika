exec {
	print 3.7E+2, _n_.
	print 3.99999999999, _n_.
	print 0.000001234, _n_.
}
