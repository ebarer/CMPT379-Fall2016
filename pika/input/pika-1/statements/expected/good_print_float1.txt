370
4
1e-006
